Movapp.xcodeproj/xcshareddata/xcschemes/MovappScreenshots.xcscheme

```diff
@@ -37,17 +37,14 @@
                BlueprintName = "MovappUITests"
                ReferencedContainer = "container:Movapp.xcodeproj">
             </BuildableReference>
-            <SkippedTests>
+            <SelectedTests>
                <Test
-                  Identifier = "MovappUITests">
+                  Identifier = "FastlaneSnapshot/testExample()">
```

Comment:
is this correct?

Reply:
This means, that only this one particular test is included in the MovappScreenshots scheme so if we will add a new test we need to add it manually for using it for screenshots.